packages/common-scripts/src/common.ts

```diff
@@ -86,27 +88,27 @@ export async function transfer(
   if (useLocktimeCellsFirst) {
     // if provider tipHeader
     if (tipHeader) {
-      [txSkeleton, deductAmount] = await lockTimePool.transfer(
+      const result = await lockTimePool.transfer(
```

Comment:
Any reason for this change?

Reply:
I forget to revert this after debug, I'll revert changes in `transfer`, it's no need to change now.